dnf-behave-tests/features/plugins-core/download.feature

```diff
@@ -0,0 +1,27 @@
+Feature: dnf download command
+
+
+Background:
+  Given I enable plugin "download"
+
+
+@bz1463723
+Scenario: Download an existing RPM in file:// mode
+  Given I use repository "dnf-ci-fedora"
+   When I execute dnf with args "download setup --destdir={context.dnf.tempdir}/downloaddir"
+   Then the exit code is 0
+    And file sha256 checksums are following
+        | Path                                                              | sha256                                                                                     |
+        | {context.dnf.tempdir}/downloaddir/setup-2.12.1-1.fc29.noarch.rpm  | file://{context.dnf.fixturesdir}/repos/dnf-ci-fedora/noarch/setup-2.12.1-1.fc29.noarch.rpm |
+
+
+  Scenario: Download an existing RPM in file:// mode with all dependencies into a --destdir
```

Comment:
Indentation alert: this line should have no indent :cat: 

Otherwise looking good.